org.gnome.baobab.json

```diff
@@ -0,0 +1,38 @@
+{
+    "app-id" : "org.gnome.baobab",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.36",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "baobab",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=host:ro",
+        "--filesystem=~/.var/app:ro",
+        "--talk-name=org.gtk.vfs.*",
+        "--system-talk-name=org.freedesktop.hostname1",
```

Comment:
What is it for, displaying local hostname?

Reply:
I do not know...  As far as I know it was in the previous pr and upstream.

Reply:
Seems so, without it, it just says "Computer". I'd drop it, it looks very uncommon to request it.

Reply:
...but the immersion :disappointed:

Reply:
I am not sure how I feel about 'just because it is uncommon'.

Reply:
Ok, so flatpak is not able to see `\` anyways. I am dropping this.